components/kyma-environment-broker/internal/orchestration/manager/manager.go

```diff
@@ -262,7 +262,7 @@ func (m *orchestrationManager) waitForCompletion(o *internal.Orchestration, stra
 			// handle the retrying ops during in progress orchestration
 			// use the existing resolved policy in op
 			numberOfNotFinished += numberOfRetrying
-			ops, err := m.factory.RetryOperations(o.OrchestrationID, o.Parameters.Strategy.Schedule, orchestration.MaintenancePolicy{}, false)
+			ops, err := m.factory.RetryOperations(o.OrchestrationID, o.Parameters.Strategy.ScheduleTime, o.Parameters.Strategy.MaintenanceWindow, orchestration.MaintenancePolicy{}, false)
```

Comment:
I would change the `factory.RetryOperations` interface to get the whole `o.Parameters` or `o.Parameters.Strategy` as one input instead of adding a new parameter